### PR TITLE
Testbranch

### PR DESCRIPTION
**Pull request by dikshatest**
@dikshatest2 This is ready for code review.

### CODE REVIEW

**Comment by dikshatest** [comment]
@dikshatest2 added more code could you pull the recent changes and review them

File: readme.md [hunk 1]
```diff
@@ -1,10 +1,5 @@
 # Hi
 # Hello
 # New branch 
-# Hello
```
Comment (dikshatest2) [inline readme.md:4]: why did you remove these?

File: test.py [hunk 2]
```diff
@@ -1 +1,2 @@
-import pygame
\ No newline at end of file
+import pygame
+x = "This is a test"
```
Comment (dikshatest2) [inline test.py:2]: change formatting

File: readme.md [hunk 3]
```diff
@@ -1,10 +1,5 @@
 # Hi
```
Comment (dikshatest2) [inline readme.md:1]: @dikshatest could have : this could look better

**Review by dikshatest2 — CHANGES_REQUESTED** [review]

File: test.py [hunk 4]
```diff
@@ -1 +1,2 @@
-import pygame
```
Comment (dikshatest2) [inline test.py:1]: you shouldn't have removed this

File: test.py (hunk 2 above)
Comment (dikshatest2) [inline test.py:2]: rhui;/;

**Review by dikshatest2 — CHANGES_REQUESTED** [review]
could you look at them please

**Review by dikshatest2 — APPROVED** [review]
All good to go ! This can be merged into master now

**Comment by dikshatest** [comment]
@bill this case is ready for release

**Comment by dikshatest** [comment]
@dikshatest deployed to .sup for testing

**Comment by dikshatest** [comment]
.sup testing complete
